internal/langserver/handlers/initialize.go

```diff
@@ -218,18 +277,18 @@ func (svc *service) Initialize(ctx context.Context, params lsp.InitializeParams)
 		}
 	}
 
-	// Walker runs asynchronously so we're intentionally *not*
-	// passing the request context here
-	walkerCtx := context.Background()
-	err = svc.walker.StartWalking(walkerCtx)
-	if err != nil {
-		return serverCaps, err
-	}
+	// // Walker runs asynchronously so we're intentionally *not*
+	// // passing the request context here
+	// walkerCtx := context.Background()
+	// err = svc.walker.StartWalking(walkerCtx)
+	// if err != nil {
+	// 	return err
+	// }
```

Comment:
I suppose we can keep starting of the walker(s) within the handler function, so this can be removed?